### ISSUE TITLE
Audio input eats a lot of CPU, does not work, sounds bad

### ISSUE THREAD

**Issue by nexus511** [nexus511/gpd-ubuntu-packages#13]
Currently audio input does not work properly/at all.

This cannot be changed in the scope of this project and needs change in the kernel.

**Comment by sobukus** [comment]
Yes, we have to hope that the state of the audio driver improves. It is rather obviously buggy.

I even managed to freeze the machine by some mp3 playback ... it suddenly looped a short piece of audio, black screen, no reaction (did not try sysrq back then). I presume it is a bit premature to open specific bugs about audio features as long as basic playback is troublesome. I even tried it directly with libasound and device hw:1,0 on the console, no desktop and no pulseaudio. No smooth playback possible. This is the same on other Cherry Trail devices, isn't it?

**Comment by nexus511** [comment]
@sobukus I have not seen freezes, but I am pretty sure that those might happen occasionally.

As there are many different problems with the audio right now, I created this collective bug. We can split this into more detailed bus as soon as the quality of the driver and pulseaudio support has been improved.

**Comment by reldred** [comment]
Hi, I just want to make one comment about sound playback; it seems as though a lot of the playback issues to me seem to boil down to it being really easy to overdrive the puny little amplifier, I tend to find around 70-75% output is the most you can drive it before playback starts to suffer. I tend to not exceed 30-50% when listening to music on headphones and audio has been perfectly smooth 

(for reference: xubuntu image, did an apt-get dist-upgrade about an hour ago from posting this)
